examples/log-ingestion/README.md

```diff
@@ -48,6 +48,12 @@ and send the processed logs to a local [OpenSearch sink](../../data-prepper-plug
 3. Run the Data Prepper docker image with the `log_pipeline.yaml` from step 2 passed in. This command attaches the Data Prepper Docker image to the Docker network `log-ingestion_opensearch_net` so that 
 FluentBit is able to send logs to the http source of Data Prepper.
 
+For Data Prepper 2.0 or above:
+```
+docker run --name data-prepper -v /full/path/to/log_pipeline.yaml:/usr/share/data-prepper/pipelines/pipelines.yaml --network "data-prepper_opensearch-net" opensearchproject/data-prepper:latest
```

Comment:
Again, please encourage the use of the same pipeline file name in Docker.

Reply:
Just noticed a small issue here, we have a sample `pipelines/pipelines.yaml` exists there in the first place. If it's not overwritten by some user-defined pipeline file with the same name, the sample pipeline file will be loaded as well and may cause confusion or conflicts.

An easy fix is to remove the sample pipelines.yaml from data prepper image.

Reply:
> An easy fix is to remove the sample pipelines.yaml from data prepper image.

I concur with this!